image-context/main.js

```diff
@@ -0,0 +1,33 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+
+const isPullRequest = github.context.eventName === "pull_request";
+const gitSha = isPullRequest
+  ? github.context.payload.pull_request.head.sha
+  : "NOT IMPLEMENTED YET";
+const gitBranch = isPullRequest
+  ? github.context.payload.pull_request.head.ref
+  : "NOT IMPLEMENTED YET";
+
+const escapedBranch = gitBranch.replaceAll(/[^A-Za-z0-9]/g, "-");
+
+const testingRegistry = "ghcr.io/wunderflats";
+const deployRegistry = "gcr.io/hosting-214616"; // GKE_PROJECT
```

Comment:
Having this value here is not super DRY. Maybe it's OK?

Alternatives:

- Set the value as an `input` to this action. This means that we have to add `with: { gke-project: gcr.io/hosting-214616 }` every time we reuse the action.
- Set the value as a project-wide environment variable, and get it from the environment.